Therify/directory
# feat: add v3 video player

# Description
Adds video player ui to align with Therify v3 styling and action.

### Next 13/Storybook bug 
`TypeError: Cannot redefine property: default`
There was a bug introduced after Next version `13.3.2` that broke Storybook builds. To be able to spin up Storybook, Next has been pinned to version`13.3.2`.

[Storybook issue](https://github.com/storybookjs/storybook/issues/23684)
# Closes issue(s)

# How to test / repro
- Spin up Storybook
- Check out "Media" -> "Video Player" -> "Player"

# Screenshots
![image](https://github.com/Therify/directory/assets/25045075/99994162-dbb2-4c8b-b3d2-e56c4dcab6f5)

# Changes include


# Checklist


# Other comments
